libethereum/EthereumCapability.cpp

```diff
@@ -545,18 +494,18 @@ vector<NodeID> EthereumCapability::selectPeers(
 }
 
 std::pair<std::vector<NodeID>, std::vector<NodeID>> EthereumCapability::randomPartitionPeers(
-    std::vector<NodeID> const& _peers, std::size_t _number) const
+    std::vector<NodeID> const& _peers, size_t _number) const
 {
-    vector<NodeID> part1(_peers);
-    vector<NodeID> part2;
+    std::vector<NodeID> part1(_peers);
+    std::vector<NodeID> part2;
 
     if (_number >= _peers.size())
         return std::make_pair(part1, part2);
 
-    std::shuffle(part1.begin(), part1.end(), m_urng);
+    shuffle(part1.begin(), part1.end(), m_urng);
```

Comment:
it seems in some random places you remove `std::` while in many others you add it

Reply:
Looks like I missed some places when I was adding them back, sorry about that! This also means that one of the #included header files does a "using namespace std"...I'll leave that alone though to keep unnecessary churn to a minimum.

Reply:
Changed unrolled!